scripts/conformance_suites/run_lrr_conformance_suite.py

```diff
@@ -0,0 +1,30 @@
+import os.path
+from unittest.mock import patch
+
+from arelle.CntlrCmdLine import parseAndRun
+from arelle.WebCache import WebCache
+
+# from https://specifications.xbrl.org/work-product-index-registries-lrr-1.0.html
+CONFORMANCE_SUITE = 'tests/resources/conformance_suites/lrr-conf-pwd-2005-06-21.zip/'
```

Comment:
nit: trailing slash initially looked like a mistake. Remove it here and append it in the replace?